src/libraries/Microsoft.Extensions.Logging.Console/tests/Microsoft.Extensions.Logging.Console.Tests/SimpleConsoleFormatterTests.cs

```diff
@@ -34,6 +32,9 @@ public void Log_WritingScopes_LogsWithCorrectColorsWhenColorEnabled(LoggerColorB
                 logger.Log(LogLevel.Information, 0, _state, null, _defaultFormatter);
             }
 
+            // For Android/iOS/tvOS/MacCatalyst default color behavior, enables the color
+            colorBehavior = colorBehavior == LoggerColorBehavior.Default && (PlatformDetection.IsAndroid || PlatformDetection.IsAppleMobile) ? LoggerColorBehavior.Enabled : colorBehavior;
```

Comment:
From documentation LoggerColorBehavior.Default means color is not enabled when the console output is redirected.
In case of Android/Apple it was enabled which I'm assuming means console output is not redirected on this platforms.

Reply:
We shouldn't be outputting color codes for Android or the Apple mobile platforms, they have no shell* and all the output gets redirected to some log file.

(*) Android kinda does via adb shell, but it's not meant for running apps.

Reply:
Is the issue here `[Console](https://source.dot.net/System.Console/A.html#f907d79481da6ba4).[IsOutputRedirected](https://source.dot.net/System.Console/A.html#876c732705561537)` behave differently between Android/Apple Mobile than other platforms? If this is the case, the line https://source.dot.net/#Microsoft.Extensions.Logging.Console/SimpleConsoleFormatter.cs,161 will need to be fixed for Android and Apple mobile I guess.

Reply:
I will revert this PR and create new one.